trlc/ast.py

```diff
@@ -480,6 +480,7 @@ def to_string(self):  # pragma: no cover
             self.__class__.__name__
 
     def ensure_type(self, mh, typ):
+        # lobster-trace: LRM.Conditional_Expression_Types
```

Comment:
I think this is the generic utility, don't link that. It takes care of null, but otherwise the callee should be linked, not that.